src/test/regress/expected/multi_view.out

```diff
@@ -206,6 +207,35 @@ SELECT count(*) FROM temp_lineitem;
   1706
 (1 row)
 
+-- can create and query materialized views
+CREATE MATERIALIZED VIEW mode_counts
```

Comment:
Yep, materialized views are really just local tables. It might be worth considering distributed materialized views at some point, along with distributed views.